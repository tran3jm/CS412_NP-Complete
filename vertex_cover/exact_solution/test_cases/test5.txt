6
A B
B A C D
C B
D B F E
F D
E D