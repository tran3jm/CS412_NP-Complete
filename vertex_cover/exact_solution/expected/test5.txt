Min number: 2
Vertices: 1 3